src/LibPQ.jl

```diff
@@ -1,13 +1,7 @@
 module LibPQ
 
-export Connection, Result, Statement
 export status, reset!, execute, clear, fetch!, prepare,
```

Comment:
These also seem like pretty generic names that could cause collisions with other packages?

Reply:
I think if someone ends up in that situation they can `import LibPQ` and use it like that. There's not really a set of things to export which isn't generic.

Reply:
Iono, I tend to err on the side of not exporting things now. This would be a lot easier if julia could just handle method definitions from different package as long as the signatures don't conflict.